modules/ppcp-applepay/src/ApplepayModule.php

```diff
@@ -0,0 +1,208 @@
+
+/**
+ * Class ApplepayModule
+ */
+class ApplepayModule implements ModuleInterface {
+	/**
+	 * {@inheritDoc}
+	 */
+	public function setup(): ServiceProviderInterface {
+		return new ServiceProvider(
+			require __DIR__ . '/../services.php',
+			require __DIR__ . '/../extensions.php'
+		);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function run( ContainerInterface $c ): void {
+		$apple_payment_method = $c->get( 'applepay.payment_method' );
+		// add onboarding and referrals hooks.
+		$apple_payment_method->initialize();
+		if ( ! $c->get( 'applepay.enabled' ) ) {
+			return;
+		}
+		if ( ! $c->get( 'applepay.server_supported' ) ) {
+			add_action(
+				'admin_notices',
+				static function () {
+					?>
+					<div class="notice notice-error">
+						<p>
+							<?php
+							echo wp_kses_post(
+								__( 'Apple Pay is not supported on this server. Please contact your hosting provider to enable it.', 'woocommerce-paypal-payments' )
+							);
+							?>
+						</p>
+					</div>
+					<?php
+				}
+			);
+
+			return;
+		}
+		if ( ! $c->get( 'applepay.merchant_validated' ) ) {
+			add_action(
+				'admin_notices',
+				static function () {
+					?>
+					<div class="notice notice-error">
+						<p>
+							<?php
+							echo wp_kses_post(
+								__( 'Apple Pay Validation Error. Please check the requirements.', 'woocommerce-paypal-payments' )
+							);
+							?>
+						</p>
+					</div>
+					<?php
+				}
+			);
+		}
+		$this->loadAssets( $c );
+		$env = $c->get( 'onboarding.environment' );
+		/**
+		 * The environment.
+		 *
+		 * @var Environment $env
+		 */
+		$is_sandobx = $env->current_environment_is( Environment::SANDBOX );
+		$this->domain_association_file( $is_sandobx );
+		$this->renderButtons( $c );
+
+		$apple_payment_method->bootstrapAjaxRequest();
+	}
+
+	/**
+	 * Returns the key for the module.
+	 *
+	 * @return string|void
+	 */
+	public function getKey() {
+	}
+
+	/**
+	 * Loads the assets.
+	 *
+	 * @param boolean $is_sandbox The environment for this merchant.
+	 */
+	protected function domain_association_file( $is_sandbox ): void {
+		if ( ! isset( $_SERVER['REQUEST_URI'] ) ) {
+			return;
+		}
+		$request_uri = (string) filter_var( wp_unslash( $_SERVER['REQUEST_URI'] ), FILTER_SANITIZE_URL );
+		if ( strpos( $request_uri, '.well-known/apple-developer-merchantid-domain-association' ) !== false ) {
+			$validation_string = $is_sandbox ? 'apple-developer-merchantid-domain-association-sandbox' : 'apple-developer-merchantid-domain-association';
+			$validation_file   = file_get_contents( __DIR__ . '/../assets/validation_files/' . $validation_string );
+			nocache_headers();
+			header( 'Content-Type: text/plain', true, 200 );
+			echo $validation_file;// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			exit;
+		}
+	}
+
+	/**
+	 * Registers and enqueues the assets.
+	 *
+	 * @param ContainerInterface $c The container.
+	 * @return void
+	 */
+	public function loadAssets( ContainerInterface $c ): void {
```

Comment:
PHPCS not running in this module? :thinking: 

It would complain about camel case.